artifact_list_builder.py

```diff
@@ -765,7 +835,6 @@ def _addArtifact(self, artifacts, groupId, artifactId, version, extsAndClass, su
         if mavenArtifact in artifacts:
             artifacts[mavenArtifact].merge(ArtifactSpec(url, artTypes))
         else:
-            logging.debug("Adding artifact %s", str(mavenArtifact))
```

Comment:
I don't like removal of this logging. It is on debug level and normally it runs with info level, so won't be visible.